lambda/supplies/supplies.js

```diff
@@ -1,23 +1,25 @@
 module.exports.handler = async (event) => {
   console.log('Request: ', event);
+  let supplyData = require('./supplies.json');
```

Comment:
Don't you want to export that out of the handle to allow reuse without having to load everytime ?

Reply:
The state is generally not persisted between lambda invocations although reading about nodejs runtime specifically, looks like in some cases it might (I never used JS in lambdas much, but for this project it looked simpler to get going with just from build and packaging perspective). I guess it makes some sense to move it out, although the load time should be minimal either way